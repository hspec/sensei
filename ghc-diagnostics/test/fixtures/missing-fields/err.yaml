version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/missing-fields/Foo.hs
  start:
    line: 3
    column: 7
  end:
    line: 3
    column: 17
severity: Warning
code: 20125
message:
- |-
  Fields of `IOError' not initialised:
    ioe_handle :: Maybe GHC.Internal.IO.Handle.Types.Handle
    ioe_type :: IOErrorType
    ioe_location :: String
    ioe_description :: String
    ioe_errno :: Maybe GHC.Internal.Foreign.C.Types.CInt
    ioe_filename :: Maybe FilePath
- |-
  In the expression: IOError {}
  In an equation for `foo': foo = IOError {}
hints: []
reason:
  flags:
  - missing-fields
