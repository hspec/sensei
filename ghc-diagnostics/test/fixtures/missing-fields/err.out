test/fixtures/missing-fields/Foo.hs:3:7: warning: [GHC-20125] [-Wmissing-fields]
    • Fields of `IOError' not initialised:
        ioe_handle :: Maybe GHC.Internal.IO.Handle.Types.Handle
        ioe_type :: IOErrorType
        ioe_location :: String
        ioe_description :: String
        ioe_errno :: Maybe GHC.Internal.Foreign.C.Types.CInt
        ioe_filename :: Maybe FilePath
    • In the expression: IOError {}
      In an equation for `foo': foo = IOError {}

